packages/components/grid/package.json

```diff
@@ -44,10 +45,12 @@
     "@sl-design-system/icon": "^1.2.0",
     "@sl-design-system/popover": "^1.2.2",
     "@sl-design-system/scrollbar": "^0.1.1",
```

Comment:
It looks like the `listbox` dependency is missing?

Reply:
Fixed